takwimu/profiles/census.py

```diff
@@ -1791,15 +1985,15 @@ def get_profile(geo, profile_name, request):
                 func = globals()[function_name]
                 data[section] = func(geo, session, country, level)
                 # get profiles for comparative geometries
-                if not data[section]['is_missing']:
+                if not data[section]['is_missing'] and not data[section]['is_missing'] is None:
```

Comment:
This `None` stuff is a ticking time bomb... Lets think and come up with an approach that will handle all these corner cases.